sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/administration/models/RuleFilter.java

```diff
@@ -27,4 +27,11 @@
         // This is intentionally left blank. This constructor exists
         // only to prevent external assemblies inheriting from it.
     }
+
+    @Override
+    public abstract boolean equals(Object obj);
+
+    @Override
+    public abstract int hashCode();
```

Comment:
As discussed offline, this is a breaking change. If customers have already extended from this abstract class without overriding `equals` and `hashCode` methods, then introducing these abstract methods here will break them.

Reply:
Agreed, makes sense closed out this PR - Let's add this to the To-Do list for major version bump